README.md

```diff
@@ -42,6 +43,14 @@ configuration parameter
 config.handlers.jira_issues.ignore = [ 'Jira', 'Github' ]
 ```
 
+Optionally you can set a timer for how long sleep prior to posting an issue to chat again.  This is accomplished by setting an expiring key in Redis. That timeout is govered by the following config
```

Comment:
Grammar here, "how long **to** sleep prior to posting". Also govered -> governed